### DISCUSSION TITLE
Support fetching a single topic from Gitlab

### DISCUSSION THREAD

**Discussion by jakejx** [magit/forge#595] [Ideas]
My company uses a monorepo setup. This means that if I fetch all topics, it would lead to a huge number of MRs being pulled into forge. I would like to be able to selectively fetch topics when I need to. When I tried to fetch an individual topic (by providing the MR number), I  received an error saying that pulling individual topics is not supported for Gitlab. Is there any way around it?

**Comment by tarsius** [comment]
> Is there any way around it?

Not without actually implementing it.

One day I will, but currently I am [very busy](https://magit.vc/stats/forge/activity.html) working on other parts of Forge.

**Reply by jakejx** [reply]
Cool, thanks for the amazing package and I'll look into implementing it!

**Comment by tarsius** [comment]
For Github this is supported now. For Gitlab it still isn't.

**Reply by tarsius** [reply]
I've added Gitlab support in 3fc32ca729c9bc0c88672f46241b4040d1684a3c.
